app/views/posts/index.html.erb

```diff
@@ -1,2 +1,27 @@
 <h1>Posts#index</h1>
```

Comment:
- [ ] I would like to recommend you remove this `<h1> ` element and make sure that your see all post pages displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) as required for this project.  for example, the user photo is square, not rounded. Please fix it :+1: